src/test/java/de/tum/in/www1/artemis/ExampleSubmissionIntegrationTest.java

```diff
@@ -205,4 +205,48 @@ public void createExampleTextAssessment() throws Exception {
         database.checkFeedbackCorrectlyStored(feedbacks, storedResult.getFeedbacks(), FeedbackType.MANUAL);
         assertThat(storedResult.isExampleResult()).as("stored result is flagged as example result").isTrue();
     }
+
+    @Test
+    @WithMockUser(value = "instructor1", roles = "INSTRUCTOR")
+    public void createExampleTextAssessment_notExistentId() throws Exception {
+        ExampleSubmission storedExampleSubmission = database.addExampleSubmission(database.generateExampleSubmission("Text. Submission.", textExercise, true));
+        database.addResultToSubmission(storedExampleSubmission.getSubmission(), AssessmentType.MANUAL);
+        final Result exampleResult = request.get("/api/exercises/" + textExercise.getId() + "/submissions/" + storedExampleSubmission.getSubmission().getId() + "/example-result",
+                HttpStatus.OK, Result.class);
+        final Set<TextBlock> blocks = ((TextSubmission) exampleResult.getSubmission()).getBlocks();
+        assertThat(blocks).hasSize(2);
+        List<Feedback> feedbacks = new ArrayList<>();
+        final Iterator<TextBlock> textBlockIterator = blocks.iterator();
+        feedbacks.add(new Feedback().credits(80.00).type(FeedbackType.MANUAL).detailText("nice submission 1").reference(textBlockIterator.next().getId()));
```

Comment:
Maybe you can use the ModelFactory (i.e. generateManualFeedback()) here and in the next test to generate feedbacks in order to remove duplicates.